code/modules/power/cable.dm

```diff
@@ -187,19 +187,19 @@ By design, d1 is the smallest direction and d2 is the highest
 		return
 
 	if(d1 == UP || d2 == UP)
-		to_chat(user, "<span class='warning'>You must cut this cable from above.</span>")
+		to_chat(user, SPAN_WARNING("You must cut this [src] from above."))
```

Comment:
`this the cable` will result from this, you want `[name]` or `cut \the [src]`.

Reply:
fixed